R/checks.R

```diff
@@ -1685,6 +1715,12 @@ checkDescription <- function(package_dir){
     }
 }
 
+checkDESCRIPTIONFile <- function(package_dir) {
+    dcf <- read.dcf(file.path(package_dir, "DESCRIPTION"))
+
+    .checkLicenseForRestrictiveUse(dcf[,"License"])
```

Comment:
`checkDESCRIPTIONFile` could go into the `checkDescription` function above to avoid re-reading the `DESCRIPTION` file. The tests for the `.checkLicenseForRestrictiveUse` function look great.